lib/resque_spec/matchers.rb

```diff
@@ -185,13 +201,13 @@ def schedule_queue_for(actual)
   chain :at do |timestamp|
     @interval = nil
     @time = timestamp
-    @time_info = "at #{@time}"
+    @time_info = " at #{@time}"
   end
 
   chain :in do |interval|
     @time = nil
     @interval = interval
-    @time_info = "in #{@interval} seconds"
+    @time_info = " in #{@interval} seconds"
```

Comment:
LOL, yea I did not completely rewrite it, so that will need to be addressed too.

Go ahead with my latest suggestion (extra comma) and I will fix that when I merge the PR. Thanks for the doing the PR.